pennylane_lightning/lightning_tensor/lightning_tensor.py

```diff
@@ -147,6 +147,12 @@ def stopping_condition(op: Operator) -> bool:
     return op.has_matrix and len(op.wires) <= 2 and op.name in _operations
 
 
+def stopping_condition_shots(op: Operator) -> bool:
+    """A function that determines whether or not an operation is supported by ``lightning.tensor``
+    with finite shots."""
+    return stopping_condition(op) or isinstance(op, qml.ops.op_math.Conditional)
+
+
```

Comment:
Do we really need this? This is usually to support mid-circuit measurements.
```suggestion
```

Reply:
Nice catch!